xml/obs_package_formats.xml

```diff
@@ -282,4 +282,124 @@ systemctl enable sshd</screen>
   <title>&obs.ai;</title>
   <remark>toms 2017-08-18: TODO What is it and what is needed</remark>
  </sect1>
+
+ <sect1>
+  <title>&flatpak;</title>
+  <para>
+  The <link xlink:href="https://flatpak.org/">Flatpak</link> format can be used
+  to generate desktop apps for linux.
+  </para>
+  <para>
+  For building an installable flatpak bundle, you need a
```

Comment:
We should capitalize project names. Look for other occurances. :slightly_smiling_face: 

```suggestion
  For building an installable Flatpak bundle, you need a
```